linkerd/namer/k8s/src/main/scala/io/l5d/k8s.scala

```diff
@@ -73,6 +73,7 @@ case class k8sConfig(
     // namer path -- should we just support a `label`?
     val path = prefix.show
     val service = client
+      .withParams(params)
```

Comment:
so, i think this will entirely replace the client's params (and i.e. wipe out the tls config, and any http-specific param settings).

This probably needs to be:
```scala
.transformed { stk => stk.withParams(stk.params ++ params) }
```